File: Assignments/Assignment_12/p8.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=0;i<row;i++){

                for(int j=0;j<row;j++){

                        if(i%2==0){

                                printf("%c\t",(96+row+j));
                        }else{

                                printf("%c\t",(96+(2*row)-j));
                        }
                }
                printf("\n\n");
        }
}

File: JAVA/8.Nested_For_1/p8.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		char ch='A';
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		for(int i=0;i<row;i++){
		
			for(int j=0;j<row;j++){
			
				System.out.print(ch+"\t");
				ch++;
			}System.out.println();
		}
	}
}
